imageio/plugins/ffmpeg.py

```diff
@@ -45,6 +45,12 @@ def get_exe():
     if exe:  # pragma: no cover
         return exe
 
+    try:
+        sp.check_output(["ffmpeg", "-version"])
+        return "ffmpeg"
+    except sp.CalledProcessError:
```

Comment:
`FileNotFoundError` is raised on 3.x (and doesn't exist on 2.x), which inherits from `OSError`.